frontend/packages/pipelines-plugin/src/components/pipelineruns/detail-page-tabs/PipelineRunDetails.tsx

```diff
@@ -1,75 +1,28 @@
-import RunDetailsErrorLog from '../logs/RunDetailsErrorLog';
-import PipelineRunVisualization from './PipelineRunVisualization';
-import TriggeredBySection from './TriggeredBySection';
-import { getPLRLogSnippet } from '../logs/pipelineRunLogSnippet';
-
+import { PipelineRunKind } from '../../../types';
+import ResultsList from '../../shared/results/ResultsList';
+import PipelineRunDetailsSection from './PipelineRunDetailsSection';
 import './TriggeredBySection.scss';
 
 export interface PipelineRunDetailsProps {
   obj: PipelineRunKind;
 }
 
 export const PipelineRunDetails: React.FC<PipelineRunDetailsProps> = ({ obj: pipelineRun }) => {
-  const { t } = useTranslation();
-  // FIXME: If they are inline resources, we are not going to render them
-  const unfilteredResources = pipelineRun.spec.resources as PipelineRunReferenceResource[];
-  const renderResources =
-    unfilteredResources
-      ?.filter(({ resourceRef }) => !!resourceRef)
-      .map((resource) => resource.resourceRef.name) || [];
-
   return (
-    <div className="co-m-pane__body odc-pipeline-run-details">
-      <SectionHeading text={t('pipelines-plugin~Pipeline Run details')} />
-      <PipelineRunVisualization pipelineRun={pipelineRun} />
-      <div className="row">
-        <div className="col-sm-6">
-          <ResourceSummary resource={pipelineRun} />
-        </div>
-        <div className="col-sm-6 odc-pipeline-run-details__customDetails">
-          <dl>
-            <dt>{t('pipelines-plugin~Status')}</dt>
-            <dd>
-              <Status
-                status={pipelineRunFilterReducer(pipelineRun)}
-                title={pipelineRunFilterReducer(pipelineRun, t)}
-              />
-            </dd>
-          </dl>
-          <RunDetailsErrorLog
-            logDetails={getPLRLogSnippet(pipelineRun, t)}
-            namespace={pipelineRun.metadata.namespace}
-          />
-          {pipelineRefExists(pipelineRun) && (
-            <dl>
-              <dt>{t('pipelines-plugin~Pipeline')}</dt>
-              <dd>
-                <ResourceLink
-                  kind={referenceForModel(PipelineModel)}
-                  name={pipelineRun.spec.pipelineRef.name}
-                  namespace={pipelineRun.metadata.namespace}
-                />
-              </dd>
-            </dl>
-          )}
-          <TriggeredBySection pipelineRun={pipelineRun} />
-          <ResourceLinkList
-            model={PipelineResourceModel}
-            links={renderResources}
-            namespace={pipelineRun.metadata.namespace}
+    <>
+      <div className="co-m-pane__body odc-pipeline-run-details">
+        <PipelineRunDetailsSection pipelineRun={pipelineRun} />
+      </div>
+
+      {pipelineRun.status?.pipelineResults && (
+        <div className="co-m-pane__body">
+          <ResultsList
+            results={pipelineRun.status?.pipelineResults}
+            resourceKind={pipelineRun.kind}
```

Comment:
```suggestion
            resourceKind={pipelineRun.label}
```

Reply:
I assumed u meant `PipelineRunModel.labe`l and updated accordingly

Reply:
Right you are, my bad! Good catch.